pkg/api/mlflow/service/run/converters.go

```diff
@@ -0,0 +1,24 @@
+package run
+
+import (
+	"fmt"
+
+	"github.com/G-Research/fasttrackml/pkg/api/mlflow/api/request"
+	"github.com/G-Research/fasttrackml/pkg/api/mlflow/dao/models"
+)
+
+// adjustSearchRunsRequestForNamespace preprocesses the SearchRunRequest for the given namespace.
+func adjustSearchRunsRequestForNamespace(ns *models.Namespace, srr *request.SearchRunsRequest) {
```

Comment:
maybe `normalise` more meaningful name?

Reply:
not clear what you mean, can you suggest a name?

Reply:
yes, yes -> `NormaliseSearchRunsRequestForNamespace`

Reply:
I don't think Normalise quite fits this situation